spec/factories/users.rb

```diff
@@ -69,7 +68,6 @@
     device_updated_at { Time.current }
     sync_approval_status { User.sync_approval_statuses[:denied] }
     email_authentications { build_list(:email_authentication, 1, email: email, password: password) }
-    organization
```

Comment:
These removals will result in less unnecessary orgs getting created throughout all our tests...I bet it shaves some time off CI.